XRTK-Core/Assets/XRTK/Inspectors/Utilities/Packages/MixedRealityPackageUtilities.cs

```diff
@@ -39,12 +41,12 @@ public static MixedRealityPackageSettings PackageSettings
         /// <summary>
         /// Is the package utility running a check?
         /// </summary>
-        public static bool IsRunningCheck { get; private set; } = false;
+        public static bool IsRunningCheck { get; private set; }
 
         /// <summary>
         /// Debug the package utility.
         /// </summary>
-        public static bool DebugEnabled { get; set; } = false;
+        public static bool DebugEnabled { get; set; } = true;
```

Comment:
I'm going to turn off the debugging once I've been able to validate that the submodule hack works